scripts/integration-tests/e2e-vue-cli.sh

```diff
@@ -30,6 +30,12 @@ node "$PWD"/../../utils/bump-babel-dependencies.js
 yarn lerna exec -- node "$PWD"/../../utils/bump-babel-dependencies.js
 yarn install
 
+if [ "$BABEL_8_BREAKING" = true ] ; then
+  # Babel 8 defaults to "default, not ie 11", but this breaks vue-cli's tests
+  # which expect different ES features to be compiled.
+  patch -p1 < "$PWD"/../../patches/vue-cli-babel-8.patch
```

Comment:
Ended up doing a patch, it feels less hacky than a regexp.

Reply:
sounds good, noticed vue-cli had one too!